src/main/java/com/comphenix/protocol/injector/netty/channel/NettyChannelInjector.java

```diff
@@ -545,13 +572,21 @@ <T> T processOutbound(T action) {
             return action;
         }
 
+        PacketType.Protocol protocol = this.getCurrentProtocol(PacketType.Sender.SERVER);
+		PacketType packetType = PacketRegistry.getPacketType(protocol, packet.getClass());
+		
+		// TODO: ignore packet or throw error?
```

Comment:
ignore, i think. maybe add debug logging if someone wants to dig into exactly why this would be happening